eng/pipelines/runtime-community.yml

```
@@ -7,6 +7,12 @@ schedules:
       include:
       - main
     always: false # run only if there were changes since the last successful scheduled run.
+  - cron: "0 4,16 * * *" # run at 4:00 and 16:00 (UTC) which is 20:00 and 08:00 (PST).
+    displayName: Runtime-community release schedule
+    branches:
+      include:
+      - release/7.0*
```

better to do it like this so we don't need to update it for 8.0 etc
```suggestion
      include:
      - release/*
     exclude:
     - release/6.0*
```

---

or actually, let's just do this in the normal schedule

---

We need some way to handle build failures for release/6.0 on ppc64el then, since that arch isn't supported in 6

---

we could just exclude release/6.0 here, the chance that something gets merged that breaks s390x is quite small

---

`release/6.0`, or `release/6.0*`?